src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -186,19 +208,19 @@ export default function CreateOfferRequest ({ history: { push } }) {
         agents={agents}
         selectedAgentId={counterpartyId}
         selectAgent={selectAgent} />
-      <Button type='submit' dataTestId='submit-button' wide variant='secondary' styleName='send-button' disabled={counterpartyId.length === AGENT_ID_LENGTH ? !isCounterpartyFound : true}>Send</Button>
+      <Button type='submit' dataTestId='submit-button' wide variant='secondary' styleName='send-button' disabled={counterpartyId.length !== AGENT_ID_LENGTH || !isCounterpartyFound || !isFormValid}>Send</Button>
```

Comment:
This is fine as it is, but I think it would be neater to have a `useEffect` that sets the value of `isFormValid` and takes into account all of the factors (the ones here and the ones already determining `isFormValid`)